android/quest/src/main/java/org/smartregister/fhircore/quest/ui/login/AccountAuthenticator.kt

```diff
@@ -78,8 +80,12 @@ constructor(
         authToken =
           try {
             tokenAuthenticator.refreshToken(refreshToken)
-          } catch (httpException: HttpException) {
-            ""
+          } catch (ex: Exception) {
+            Timber.e(ex)
+            when (ex) {
+              is HttpException, is UnknownHostException -> ""
+              else -> throw ex
```

Comment:
Would this be handled elsewhere? 
Looks like there's potential for a crash if we rethrow an Exception. 🤔

Reply:
true but I think that's ok, that not this functions responsibility